src/OrbitGl/include/OrbitGl/BasicPageFaultsTrack.h

```diff
@@ -46,9 +42,10 @@ class BasicPageFaultsTrack : public LineGraphTrack<kBasicPageFaultsTrackDimensio
   // unknown type.
   [[nodiscard]] Track::Type GetType() const override { return Track::Type::kUnknown; }
 
+  [[nodiscard]] size_t GetDimension() const { return series_.GetDimension(); }
```

Comment:
I'm wondering if this member function should be in the base class that also contains `series_`. WDYT?